project/BuildDependencies/DownloadBuildDeps.bat

```diff
@@ -29,14 +37,26 @@ md %TMP_PATH%
 cd scripts
 
 SET FORMED_OK_FLAG=%TMP_PATH%\got-all-formed-packages
-REM Trick to preserve console title
+:: Trick to preserve console title
```

Comment:
Should use "REM" for comments. Using "::" for comments can cause issues.

See: http://www.robvanderwoude.com/comments.php

The double-colon is not documented as a comment command, it is a special case of a label that acts as a comment. Meaning, it is NOT supported and can change in new builds of windows. So why use it when REM is the correct method for comments.

And the "slow" issue mentioned above is not really an issue anymore. Perhaps back in Win95 days.